wicketd/src/rss_config.rs

```diff
@@ -164,6 +164,17 @@ impl CurrentRssConfig {
             bootstrap_ips.push(ip);
         }
 
+        // LRTQ requires at least 3 sleds
+        let trust_quorum_peers: Option<
+            Vec<bootstrap_agent_client::types::Baseboard>,
+        > = if known_bootstrap_sleds.len() >= 3 {
+            Some(
+                known_bootstrap_sleds.keys().cloned().map(Into::into).collect(),
+            )
+        } else {
+            None
+        };
```

Comment:
Should we be doing this? Couldn't this just be a configuration option through wicket, that a user could set explicitly?

I have a small amount of fear for "auto-configured options that turn off security features" -- if we set this explicitly, and threw an error in the one/two sled cases ("You enabled trust quorum, but do not have sufficient sleds to have a quorum") that actually seems preferable to quietly shutting it off.

Reply:
Good catch! I had a similar thought when writing this code. The only thing that made me confident was the fact that we aren't going to sell a rack to someone with two sleds. Adding a configuration through wicket isn't a bad idea, but its a touch more work right now, that I don't think is strictly necessary. Mind if I open an issue for it and mark it security? I can target it for MVP.

Reply:
I am fine deferring this, but I'd like to remove the comments scattered through the code about "we do this special thing if there are more than three sleds". That seems like a policy decision, and one we should keep in a single spot.

Reply:
Thinking more about this, I don't think it is correct to give the user the choice to disable trust quorum if they have 3 sleds or more. We can alert them that trust quorum is disabled if they try to configure a rack with only 1 or 2 sleds, but they should not be able to disable trust quorum. Since that's the case, I think the current logic is correct, we just must add a log message and UI warning. I will add a log message and open an issue for a UI warning.

Reply:
I created an [issue](https://github.com/oxidecomputer/omicron/issues/3690)

Reply:
Logging added in 1ad6da263